### ISSUE TITLE
Low mAP of yolo-tiny-prn.

### ISSUE THREAD

**Issue by ChenCong7375** [AlexeyAB/darknet#4091]
I trained yolo-tiny-prn.cfg on VOC dataset with yolo-tiny-conv.13 as pretrain model.
I got 66%mAP on VOC of yolov3-tiny.cfg
But yolov3-tiny-prn.cfg got a lower mAP .
Have you ever trained it on your own dataset?

**Comment by AlexeyAB** [comment]
What mAP did you get on?
1. yolov3-tiny.cfg
2. yolo-tiny-prn.cfg
3. yolov3-tiny-prn.cfg

**Comment by ChenCong7375** [comment]
@AlexeyAB 
yolov3-tiny.cfg:320*320,66%mAP
yolov3-tiny-prn.cfg:320*320,60%mAP

**Comment by AlexeyAB** [comment]
How many iterations did you train both cfg-files?

**Comment by ChenCong7375** [comment]
50000 iterations，for 20 classes in VOC dataset

**Comment by WongKinYiu** [comment]
i think the reason is that u use the coco pretrained model on yolo-v3-tiny instead of yolo-v3-tiny-prn.
i use prn.conv.15 instead of tiny.conv.13, and the results are as follows:
![image](https://user-images.githubusercontent.com/12152972/66888596-cadb7100-f011-11e9-9c95-5e5e98450e28.png)

**Comment by ChenCong7375** [comment]
@WongKinYiu thanks very much for your repo.

**Comment by ChenCong7375** [comment]
@WongKinYiu So you used prn.conv.15 and get 64% mAP? 
But I got 68% mAP… so can you share your chart?

**Comment by ChenCong7375** [comment]
![prn416](https://user-images.githubusercontent.com/32812662/67173576-b4f2f500-f3f1-11e9-9d3d-e37323ab547e.png)
@WongKinYiu

**Comment by WongKinYiu** [comment]
i do not save the chart.
and i trained the model in march, there are so many improvement of this repo in these days.

however, 68% mAP is better than 64%, it is a good news.

**Comment by samohadid** [comment]
hi ,
can you give me a link to download  prn.conv.15 pre-trained model ? 
thank you very much

**Comment by developer0hye** [comment]
@WongKinYiu 

Thank you for experiments!

I have two quesetions.

1. What does W/O COCO model mean?

training from imagenet pretrained model? or training from scratch?

2. Normalization method for preprocessing

Did you normalized the input image by mean and std value?

If you did, what do you use mean and std from imagenet or voc dataset?

**Comment by WongKinYiu** [comment]
>What does W/O COCO model mean?

w\o = without, without coco pretrain.

> training from imagenet pretrained model? or training from scratch?

from imagenet pretrain

> Normalization method for preprocessing

/255.

**Comment by developer0hye** [comment]
@WongKinYiu 
Thanks!

**Comment by developer0hye** [comment]
@WongKinYiu 

I have new one question.

How do you normalize image to train the model on Imagenet? 

same with /255.?
